File: python/raindrops/raindrops.py
def convert(number):
    d = {3: "Pling", 5: "Plang", 7: "Plong"}
    output = "".join(map(lambda n:  d[n] if number % n == 0 else "", d.keys()))
    return str(number) if output == "" else output

File: clojure/raindrops/src/raindrops.clj
(ns raindrops
  (:require [clojure.string :as str]))

(defn convert [n] 
  (let [data {3 "Pling", 5 "Plang", 7 "Plong"} 
        output (map #(if (= (mod n (first %)) 0) (last  %) "") data)]
    (if (= (str/join output) "") (str n) (str/join output))))
